docs/DeveloperGuide.md

```diff
@@ -355,7 +355,7 @@ This feature allows telemarketers to edit data fields at any point in time if th
 
 Given below is an example usage scenario and how the edit mechanism behaves at each step.
 
-Step 1. The user launches the application and realizes that person 1 needs to be edited. He/she executes the command `edit 1 name/Will age/20` in order to edit the first person in the address book.
+Step 1. The user launches the application and realizes that person 1 needs to be edited. He/she executes the command `edit 1 n/Will age/20` in order to edit the first person's name and age in the address book.
```

Comment:
use contacts. 
and also 'realizes that the first contact' or 'realizes that the details of the first contact need to be'